src/common/numbers.js

```diff
@@ -1,4 +1,6 @@
-export function countToLocaleString(num) {
+const countToLocaleString = (num) => {
   if (!num?.toLocaleString || typeof num !== 'number') return num;
   return num.toLocaleString();
-}
+};
+
+export { countToLocaleString };
```

Comment:
auto-formating?

Reply:
Yes, I think that is what happened there and I also changed that function to an arrow function